siac/bashcomplcmd.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "github.com/spf13/cobra"
+
+var (
+	bashcomplCmd = &cobra.Command{
+		Use:   "bash-completion [path]",
+		Short: "Creates bash completion file.",
+		Long: "Creates a bash completion file at the specified " +
+			"location.\n\n" +
+
+			"Note: Bash completions will only work with the " +
+			"prefix with which the script is created (e.g. " +
+			"`./siac` or `siac`).\n\n" +
+
+			"Once created, the file has to be moved to the bash " +
+			"completion script folder - usually " +
+			"`/etc/bash_completion.d/`.",
```

Comment:
this is fine, but typically we use raw string literals (` `` `) for multi-line strings